cg-spec/editors_draft.html

```diff
@@ -224,16 +227,16 @@ <h2>Background and Motivation</h2>
+    <p>Sometimes, we want the subject or object of a triple to refer to another triple. For example, the statement "according to employee22, employee38 has a jobTitle of 'Assistant Designer'" can be modeled as a triple with "according to" as its predicate, employee22 as its object, and another triple as its subject, namely the triple "employee38 has a jobTitle of 'Assistant Designer'". This use of a triple as the subject or object resource of another triple so that we can say things about that triple is known as <a href='https://www.w3.org/TR/rdf11-mt/#reification'>reification</a>. The concept of reification has always been part of RDF, but expressing it in most RDF concrete syntaxes such as Turtle and N-Triples, as well as processing or querying it with SPARQL, has been verbose and cumbersome [[RDF-STAR-FOUNDATION]]. A notable exception is RDF/XML which provides the <a href="https://www.w3.org/TR/rdf-syntax-grammar/#section-Syntax-reifying">ID</a> attribute as syntactic sugar for the standard reification quadlet <span class=""heavy">[Note: see my <a href="https://github.com/w3c/rdf-star/pull/171#issuecomment-854648532">comment on issue #171</a> for why I think this sentence is important].</span></p>
+
+    <p>RDF-star however deviates from RDF standard reification in fundamental ways concerning both <a href="#occurrences">abstract syntax</a> and <a href="#referential-opacity">semantics</a>. It thereby enables new use cases like Explainable AI but requires extra effort when used to model common idioms like provenance annotations, n-ary relations and Property Graph-style attributed relations.</p>
 
     </section>
 
     <section class="informative">
       <h2>Overview</h2>
 
-<p>This specification describes RDF-star, an extension of RDF's conceptual data model and concrete syntaxes, providing a more compact form of reification. This model and syntaxes enable the creation of concise triples that reference other triples as subject and object resources.
-Triples that include a triple as a subject or an object are known as <a>RDF-star triples</a>. The following dataset shows the example RDF-star triples from above using the <a href="#turtle-star">Turtle-star</a> syntax, which uses double angle brackets to enclose a triple serving as a subject or object resource:</p>
+    <p>This specification describes RDF-star, an extension of RDF's conceptual data model and concrete syntaxes, providing a more compact form of reification. This model and syntaxes enable the creation of concise triples that reference the syntactic representation of other triples as subject and object resources. Triples that include a triple as a subject or an object are known as <a>RDF-star triples</a>. The following dataset shows the example RDF-star triples from above using the <a href="#turtle-star">Turtle-star</a> syntax, which uses double angle brackets to enclose a triple serving as a subject or object resource:</p>
```

Comment:
This addition is ambiguous and possibly misleaning. By "syntactic representation", do you mean "abstract syntax" or "concrete syntax"?

If you mean "abstract syntax", then this is redundant, because by definition, triples are elements of the abstract syntax.

If you mean "concrete syntax", then this is wrong. "<< :alice a :Person >>" and "<< :alice rdf:type :Person >>" are two different pieces of Turtle (abstract syntax), but represent the same triple (assuming the usual prefix declaration for `rdf:` of course).

Reply:
Redundancy is not necessarily a bad thing. Nothing is harder to read than an overly terse text. Additionally this is the overview section and the first mention of embedded triples (not by that name, but all the same). 
That said, can you make a better proposal, instead of "syntactic representation"? I'm certainly not married to that expression but I want to get across an early hint that embedded triples are different from normal triples in the way that later the section on "Referential Opacity" explains in detail.